components/api_server/src/example-reports/example-report.json

```diff
@@ -107,7 +107,7 @@
                             "type": "testng",
                             "parameters": {
                                 "url": "http://testdata:8000/reports/testng/testng-results.xml",
-                                "landing_url": "http://testdata:8000/reports/testng/testng-results.xml",
+                                "landing_url": "http://localhost:8000/reports/testng/testng-results.xml",
```

Comment:
This means the example report won't work when Quality-time is deployed to a Docker (or Kubernetes) host, while that is the scenario we have been supporting thus far with the example report.

Reply:
I'm not sure, I brought the test data in line with all other cases. We specify `testdata` as host for "url" and `localhost` as host for "landing_url"